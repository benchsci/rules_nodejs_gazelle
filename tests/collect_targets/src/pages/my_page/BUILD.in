# gazelle:js_collect_all