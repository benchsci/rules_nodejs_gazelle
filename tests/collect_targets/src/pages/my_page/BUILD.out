load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:js_collect_all

ts_project(
    name = "my_page",
    srcs = [
        "index.ts",
        "module_file.ts",
    ],
)
